src/components/Slider/Slider.js

```diff
@@ -94,7 +96,11 @@ export class RawSlider extends Component {
       pips === false ? [] : this.computeDefaultPitPoints({ min, max });
 
     return (
-      <div className={this.isDisabled ? 'ais-range-slider--disabled' : ''}>
+      <div
+        className={cx(suit(), cssClasses.root, {
```

Comment:
`cssClasses.root` already contains `suit()`.

I wouldn't have used the suit in here, but rather pass only the cssClasses as the source for anything.